simple_youtube_api/Video.py

```diff
@@ -157,15 +157,21 @@ def set_public_stats_viewable(self, viewable: bool):
         else:
             raise Exception("Not a valid status: " + str(viewable))
 
-    # TODO enforce (YYYY-MM-DDThh:mm:ss.sZ) format
-    def set_publish_at(self, time: str):
+    #@video_status_set
+    def set_publish_at(self, publish_at):
```

Comment:
We should allow passing in a string or a datetime object. Also have the type for the inputs. Ex. time: str